### PR TITLE
FEAT: Add option to supply a random seed (issue #153)

### LINKED ISSUE #153: Option to supply a random seed
Given the discussion in #147, I would like to discuss the way how to give the user the option to supply a random seed; we better fix a common interface and build a module that provides that interface.

A simple form would be something like this:

``` python
def some_function(main_arguments, random_state=None):
    """
    Parameters
    ----------
    ...

    random_state : scalar(int) or np.random.RandomState

    """
    if random_state is None or isinstance(random_state, int):
        random_state_ = np.random.RandomState(random_state)
    elif isinstance(random_state, np.random.RandomState):
        random_state_ = random_state
    else:
        raise ValueError('some message')

    # Generate a random number
    u = random_state_.random_sample()
```

For example, `scikit-learn` has a [guideline on random numbers](http://scikit-learn.org/stable/developers/contributing.html#random-numbers) and a common routine [check_random_state](https://github.com/scikit-learn/scikit-learn/blob/bb39b49/sklearn/utils/validation.py#L503) that converts the input into a `np.random.RandomState` instance.

### PR DESCRIPTION
**Pull request by QBatista**
Adds a random_state option to the remaining files mentioned in issue #153. I haven't added a test for lqnash.py yet -- @cc7768 did you have something in mind to add to this file? 

Close #153

### CODE REVIEW

**Comment by oyamad** [comment]
@QBatista Thanks.

* The first line of the docstring of `random_state` should be

  ```
  random_state : int or np.random.RandomState, optional
  ```

  (See https://github.com/QuantEcon/QuantEcon.py/issues/153#issuecomment-107135541, https://github.com/QuantEcon/QuantEcon.py/issues/153#issuecomment-107324987.)

**Comment by mmcky** [comment]
Thanks @QBatista this is looking good. I will leave this PR open a couple of days in case others wish to review / comment.
